checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py

```diff
@@ -78,33 +76,16 @@ async def report_results_async(
             get_default_post_headers(bc_platform_integration.bc_source, bc_platform_integration.bc_source_version),
             {"Authorization": bc_api_key},
         )
+        url = f"{bc_integration.api_url}{self.vulnerabilities_base_path}/results"
+
+        logging.info(f"[twistcli](report_results_async) - reporting results to the server for the file \'{file_path}\'")
+        status: int = await aiohttp_client_session_wrapper(url, headers, payload)
+
+        if status == 1:
+            logging.error(f"[twistcli](report_results_async) - Failed to send report for package file {file_path}"
+                          f"\nerror message is apear above")
```

Comment:
Don't we need to catch the possible exception?

Reply:
in case we got status = 1, there are no exception that was raised

Reply:
dicussed